pkg/apis/internal.acorn.io/v1/appinstance.go

```diff
@@ -3,13 +3,16 @@
 package v1
 
 import (
+	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
 type AppInstanceCondition string
 
 var (
 	AppInstanceConditionDefined    = "defined"
+	AppInstanceConditionDefaults   = "defaults"
```

Comment:
This looks like it is unused.

Reply:
This is used by the `pkg/controller/defaults.Calculate()` function to set the status.